Localization/Config/en-US_Mods.ImproveGame.Configs.hjson

```diff
@@ -752,7 +751,7 @@ ImproveConfigs: {
 
 	PylonTeleNoBiome: {
 		Label: "[centeritem:TeleportationPylonMushroom] Biome-independent Pylon Teleportation"
-		Tooltip: '''"It is if you say so."'''
+		Tooltip: "'Should I command such...'"
```

Comment:
Is this a better translation? I'd say leave it so or `"'If you say so.'"`.

Reply:
From what I see, the Chinese uses some sort of an idiom, so I thought it would be logical to also use an idiom. In this case this is "Should I command such, the tides shall wave". I can change it to the full quote

Reply:
May I ask for the source?

Reply:
Here, I retaliate. I asked about some idioms from a friend group and that's where I got it. Should've actually searched for it myself, because seemingly it's not an existing idiom. Apologies